python/ray/serve/handle.py

```diff
@@ -744,6 +774,30 @@ async def __call__(self, name: str) -> str:
         assert response.result() == "Hello world!"
     """
 
+    def init(
```

Comment:
shall we leave this a private API for now given that there are no public arguments?

Reply:
makes sense, I'll leave it as `_init` for now